src/controller/python/test/test_scripts/base.py

```diff
@@ -429,35 +407,36 @@ def TestFailsafe(self, nodeid: int):
             pass
```

Comment:
nit, again, I know that this is outside your PR but can you update this catch of all exceptions to catching the excepted exception (in code above)?

If that is too much a TODO with a link to github issue number is good enough for me.

Reply:
Yeah looked at that too and thought about it :smile: 

Did just that now.